src/display/canvas.js

```diff
@@ -1598,6 +1599,8 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       var defaultVMetrics = font.defaultVMetrics;
       var widthAdvanceScale = fontSize * current.fontMatrix[0];
 
+      // console.log(glyphs, fontSize, current.fontMatrix[0], fontSizeScale);
```

Comment:
The changes in this file should be removed (debug print and irrelevant whitespace change).